src/services/GenerateVerifiableCredential.ts

```diff
@@ -106,30 +106,30 @@ export class GenerateVerifiableCredential {
   	}
   	return 0;
   }
-	
-	/**
-	 * IF ID_DOCUMENT_FACE_MATCH has recommendation of "APPROVE" a score of 3 will be given else 0
-	 * 
-	 * Confulence Link: https://govukverify.atlassian.net/wiki/spaces/FTFCRI/pages/3545792513/Draft+-+Generating+Verification+from+Yoti+Results
-	**/
-  private calculateVerificationProcessLevel(faceMatchCheck: string): number {
-  	return faceMatchCheck === YotiSessionDocument.APPROVE ? 3 : 0;
+
+  /**
+   * IF ID_DOCUMENT_FACE_MATCH has recommendation of "APPROVE" and validityScore isn't 0, a score of 3 will be given else 0
+   *
+   * Confulence Link: https://govukverify.atlassian.net/wiki/spaces/FTFCRI/pages/3545792513/Draft+-+Generating+Verification+from+Yoti+Results
+   **/
+  private calculateVerificationProcessLevel(validityScore: number, faceMatchCheck: string): number {
+  	return faceMatchCheck === YotiSessionDocument.APPROVE || validityScore != 0 ? 3 : 0;
```

Comment:
just to check: should this be logical OR or AND ?

Reply:
yeh seems some of my local changes got reverted accidentally...thanks for spotting, updated